utils/slither_format/format_constable_states.py

```diff
@@ -0,0 +1,30 @@
+import re, logging, sys
+from slither.utils.colors import red, yellow, set_colorization_enabled
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('Slither.Format')
+set_colorization_enabled(True)
+
+class FormatConstableStates:
+
+    @staticmethod
+    def format(slither, patches, elements):
+        for element in elements:
+            FormatConstableStates.create_patch(slither, patches, element['source_mapping']['filename_absolute'], element['name'], "constant " + element['name'], element['source_mapping']['start'], element['source_mapping']['start'] + element['source_mapping']['length'])
```

Comment:
Consider using multiple lines declaration (this comment is true for most of the files of the PR).
We should follow as best as we can the [pep8 max legnth](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)